stop_and_wait_plugin/src/stop_and_wait_plugin.cpp

```diff
@@ -216,6 +216,14 @@ std::vector<carma_planning_msgs::msg::TrajectoryPlanPoint> StopandWait::trajecto
   for (size_t i = 0; i < points.size(); i++)
   {
     carma_planning_msgs::msg::TrajectoryPlanPoint tpp;
+
+    if (times[i] != 0 && !std::isnormal(times[i]) && i != 0)
+    {  // If the time
+      RCLCPP_WARN_STREAM(rclcpp::get_logger("stop_and_wait_plugin"),"Detected non-normal (nan, inf, etc.) time."
+        "This happens due to  dv = dx/0.0 = inf speed. Incrementing time by arbitrary large number to achieve stopping behavior");
+      times[i] = times[i - 1] + 3600.0; //assigning 10hr difference between what is normally 0.1-0.2 sec
```

Comment:
It sounds to me that the times should never be modified. If anything is modified it should be the points. The issue of infinite acceleration arises when two points are the same but the initial point has a non zero velocity. Am I missing something here. The times should always be separated by 0.1 right?

Reply:
Time should still be modified because the list would end up having `inf` values, which is not usable down the line. However, we can have it separated by 0.1 sec, I think that's no problem.